### PR TITLE
docs: Update readme windows

### PR DESCRIPTION
**Pull request by vsiravar**
Issue #, if available:

*Description of changes:*
- Update README to add instructions for installing and running finch on windows. 
*Testing done:*

- [X] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,12 +37,22 @@ To get started with Finch on macOS, the prerequisites are:
...
 
-#### Installing Finch via [brew](https://brew.sh/)
+##### Installing Finch via [brew](https://brew.sh/)
 
 ```sh
 brew install --cask finch
 ```
 
+#### windows
+To get started with Finch on macOS, the prerequisites are:
```
Comment (ginglis13) [inline README.md:47]:
```suggestion
To get started with Finch on Windows, the prerequisites are:
```

**Review by ginglis13 — COMMENTED** [review]
nice! just the nit I left, and let's capitalize windows -> Windows where applicable 🚀

File: README.md (hunk 1 above)
Comment (vsiravar) [reply README.md]: Good catch!

File: README.md [hunk 2]
```diff
@@ -35,12 +37,22 @@ To get started with Finch on macOS, the prerequisites are:
...
 
 ```sh
 brew install --cask finch
 ```
 
+#### Windows
+To get started with Finch on Windows, the prerequisites are:
+
+* Windows 10 version 2004 and higher (Build 19041 and higher)
+* Intel system for Windows
```
Comment (pendo324) [inline README.md:50]: I think "AMD64 based Windows systems" is more accurate since AMD CPUs are also compatible

File: README.md [hunk 3]
```diff
@@ -35,12 +37,22 @@ To get started with Finch on macOS, the prerequisites are:
...
+* WSL 2 installed (`wsl --install`)
+
+Download a msi installer from the [project's GitHub releases](https://github.com/runfinch/finch/releases) page, and once downloaded double click and follow the directions.
```
Comment (pendo324) [inline README.md:53]: nit
```suggestion
Download an MSI installer from the [project's GitHub releases](https://github.com/runfinch/finch/releases) page, and once downloaded double click and follow the directions.
```

**Review by pendo324 — APPROVED** [review]
Looks good overall, left a few minor comments

File: README.md (hunk 2 above)
Comment (vsiravar) [reply README.md]: Good point, updated.

**Pull request merged**